packages/mc-html-template/lib/load-env.js

```diff
@@ -1,16 +1,53 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
 const fs = require('fs');
 
 // Keep a reference to the loaded config so that requiring the module
 // again will result in returning the cached value.
 let loadedEnv;
 
-module.exports = configPath => {
-  if (loadedEnv) return loadedEnv;
+const START_OF_ENV_PLACEHODER = 'env:';
+const isEnvVariablesPlaceholder = valueOfEnvConfig =>
+  valueOfEnvConfig.startsWith(START_OF_ENV_PLACEHODER);
+const substitutePlaceholder = valueOfEnvConfig => {
+  const nameOfEnvVariable = valueOfEnvConfig.replace(
+    START_OF_ENV_PLACEHODER,
+    ''
+  );
+
+  const valueOfEnvVariable = process.env[nameOfEnvVariable];
+
+  if (!valueOfEnvVariable) {
+    throw new Error(
+      `Missing '${nameOfEnvVariable}' specified in config as 'env:${nameOfEnvVariable}'.`
+    );
+  }
+
+  return valueOfEnvVariable;
+};
+const substituteEnvVariablePlaceholders = config => {
```

Comment:
We go entries and fromEntries. Essentially a mapValues but without the dependency.